### PR TITLE
Create UWMadison_Payseur.yaml

### PR DESCRIPTION
**Pull request by showmic09**

### CODE REVIEW

File: projects/UWMadison_Payseur.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Department: Genetics
+Description: Extending access to OSPool long-term following OSG User School
```
Comment (matyasselmeci) [inline projects/UWMadison_Payseur.yaml:2]: Is this the description you want to use? It doesn't say much about what their project is about...

Comment (showmic09) [reply projects/UWMadison_Payseur.yaml:2]: @matyasselmeci I can copy something from their lab's website-https://payseur.genetics.wisc.edu/research/ or ask the user Emma to fill in with a better sentence.

Comment (showmic09) [reply projects/UWMadison_Payseur.yaml:2]: @ChristinaLK let me know which of the above options that you prefer?

Comment (brianhlin) [reply projects/UWMadison_Payseur.yaml:2]: @showmic09 I would ask the user for a description, or if there's something that you can take off of the lab's website that you linked

Comment (brianhlin) [reply projects/UWMadison_Payseur.yaml:2]: Ping @showmic09

Comment (showmic09) [reply projects/UWMadison_Payseur.yaml:2]: @brianhlin Totally forgot about that. Pinged the user and if she doesn't reply by Tuesday, I will add a sentence from their lab webpage.

Comment (showmic09) [reply projects/UWMadison_Payseur.yaml]: @brianhlin The user didn't reply so I made up a sentence based on their research work

**Pull request merged**